### PR TITLE
Sharing Gooey scheme for Carthage compatibility.

### PR DESCRIPTION
**Pull request by powerje**
Forgot to do this earlier and obviously didn't actually test Carthage support 😅

### CODE REVIEW

File: Gooey.podspec [hunk 1]
```diff
@@ -1,6 +1,6 @@
 Pod::Spec.new do |s|
   s.name                  = 'Gooey'
-  s.version               = '1.0.1'
+  s.version               = '1.0.2'
```
Comment (rwachowski) [inline Gooey.podspec]: Does it make sense for this to be a podspec version bump? This only affects Carthage and testing which aren't in the scope of the pod.

**Review by rwachowski — COMMENTED** [review]
One comment about the podspec version change

File: Gooey.podspec (hunk 1 above)
Comment (powerje) [reply Gooey.podspec]: Good point!

**Comment by powerje** [comment]
Aaand #9 shared this scheme already 😅

**Pull request closed without merging**